corehq/apps/domain/templates/domain/renew_plan.html

```diff
@@ -7,18 +7,28 @@
 {% block plan_breadcrumbs %}{% endblock %}
 
 {% block form_content %}
+  {% requirejs_main 'accounting/js/renew_plan_selection' %}
+
   <p class="lead text-center">
     {% blocktrans with plan.name as p %}
-      You are renewing your <strong>{{ p }} Edition</strong> subscription.
+      You are renewing your <strong>{{ p }}</strong> subscription.
     {% endblocktrans %}
   </p>
-  {% include 'accounting/partials/confirm_plan_summary.html' %}
 
   <form class="form"
         method="post"
         action="{% url 'domain_subscription_renewal_confirmation' domain %}">
     {% csrf_token %}
 
+    {% if request|toggle_enabled:'SELF_SERVICE_ANNUAL_RENEWALS' and is_self_renewable_plan %}
+      {% initial_page_data 'renewal_choices' renewal_choices %}
```

Comment:
it's not necessary to wrap the `initial_page_data` blocks in the if statement for toggles. I suppose this is fine for now, but best practice is to put `initial_page_data` at the beginning of the main content block of that template